File: 2022.1/Programação Orientada a Objetos/lista1java/Ex6.java
public class Ex6{
    public static void main(String args[]) throws Exception{
        int x = EntradaTeclado.leInt();

        int div;

        for(int i = x-1; i >= 2; i--){
            div = 0;
            for(int j = 1; j <= i; j++){
                if(i%j == 0) div++;
            }

            if(div == 2){
                System.out.println("O primeiro primo menor que " + x + " é: " + i);
                return;
            }
        }
    }
}

File: 2022.1/Programação Orientada a Objetos/lista1python/Ex6.py
x = int(input())

for i in range(x-1,1,-1):
    div = 0
    for j in range(1,i+1):
        if i%j == 0:
            div += 1
    if div == 2:
        print("O primeiro primo menor que", x, "é:", i)
        exit()